smp/os_management.py

```diff
@@ -86,18 +86,18 @@ class TaskStatistics(BaseModel):
 
     The unit is system dependent and in case of Zephyr this is number of 4 byte words.
     """
-    stksize: int
+    stksize: int = Field(validation_alias=AliasChoices('stksize', 'stksiz'))
```

Comment:
I'm guessing that the wrong alias was included to get the tests to pass. If that's the case I think that these "wrong" tests should get fixed. I don't see any history of "stksize" being valid.